src/index.ts

```diff
@@ -1,60 +1,99 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+/* eslint-disable @typescript-eslint/no-unsafe-call */
+/* eslint-disable @typescript-eslint/no-unsafe-member-access */
+/* eslint-disable @typescript-eslint/no-var-requires */
 /**
  * @Author: Rostislav Simonik <rostislav.simonik@technologystudio.sk>
  * @Date: 2020-06-10T16:06:00+02:00
  * @Copyright: Technology Studio
 **/
 
-module.exports = {
-  overrides: [
-    {
-      files: ['*.ts', '*.tsx'],
-      extends: [
-        'plugin:@typescript-eslint/recommended-type-checked',
-        'plugin:@typescript-eslint/stylistic-type-checked',
-        'standard-with-typescript',
-        'plugin:jest/recommended',
-        'plugin:eslint-comments/recommended',
-        'plugin:import/recommended',
-        'plugin:import/typescript',
-        ...[
-          './configs/typescript',
-          './configs/standard',
-          './configs/import',
-          './configs/eslint-comments',
-        ].map(relativePath => require.resolve(relativePath)),
-      ],
-      parser: '@typescript-eslint/parser',
-      plugins: [
-        '@typescript-eslint',
-      ],
+import type { Linter } from 'eslint'
+import globals from 'globals'
+
+import { typescriptRules } from './configs/typescript'
+import { standardRules } from './configs/standard'
+import { importRules } from './configs/import'
+import { eslintCommentsRules } from './configs/eslint-comments'
+
+const jestPlugin = require('eslint-plugin-jest')
+const { FlatCompat } = require('@eslint/eslintrc')
+const Parser = require('@typescript-eslint/parser')
+
+const compat = new FlatCompat({
+  baseDirectory: __dirname,
+})
+
+const typescriptConfig = {
+  files: ['**/*.ts', '**/*.tsx'],
+  languageOptions: {
+    globals: {
+      ...globals.jest,
```

Comment:
maybe just should be always applied only to `['__test__/**/*.ts', '__test__/**/*.tsx']` files